ADAL/src/request/ADWebRequest.h

```diff
@@ -54,6 +54,7 @@ typedef void (^ADWebResponseCallback)(NSMutableDictionary *);
 }
 
 @property (strong, readonly, nonatomic) NSURL               *URL;
+@property (copy, nonatomic)             NSMutableDictionary *headers;
```

Comment:
This line was removed here (https://github.com/AzureAD/azure-activedirectory-library-for-objc/pull/946/files#diff-298c801c6f46cf1f876980415d02c59bL57), I add it back.

For reason behind "copy", my guess is: If the `NSMutableDictionary` being passed in is somehow corrupted later due to memory corruption issue, copying it can isolate the request from being affected.

But just a guess, may need opinion from @RPangrle :)